Sources/Database/Live.swift

```diff
@@ -286,7 +286,7 @@ extension Client {
           """
           SELECT *
           FROM "users"
-          WHERE lower("rss_salt") = lower(\(bind: salt))
+          WHERE "rss_salt" = \(bind: salt)
```

Comment:
the `lower` stuff isn't necessary anymore?

Reply:
Done